docs/reference/Admin_Actions.md

```diff
@@ -4,6 +4,30 @@ When you consider that most backend services are a black box of code and decisio
 
 With that said this document will serve as the ongoing history of administrative actions that must be taken against the backend.
 
+## 2023 - April 15
+
+### Uber-Juno
+
+The [Uber-Juno](https://github.com/JunoLab/uber-juno) package uses an older unsupported coding style, sometimes called JavaScript Sloppy Mode, that's no longer supported in newer versions of NodeJS. Meaning the package crashes as soon as installed, and cannot be used without the user manually editing the source code.
+
+One of the Pulsar contributors has [submitted a fix](https://github.com/JunoLab/uber-juno/pull/85) for this issue, although it has not been updated on the Pulsar Package Registry.
```

Comment:
```suggestion
[A fix](https://github.com/JunoLab/uber-juno/pull/85) from one of the Pulsar contributors has been merged, although it has not been updated on the Pulsar Package Registry.
```

Owing to the mention of "was accepted" in the section for Hydrogen (L22 in the master version)